src/site/components/fullwidth_banner_alerts.drupal.liquid

```diff
@@ -0,0 +1,76 @@
+{% for entity in bannerAlert.entities %}
+  {% assign dismissStatus = entity.fieldAlertInheritanceSubpages %}
+  {% assign alertType = entity.fieldAlertType %}
+
+  {% if alertType = "information" %}
+    {% assign alertType = "info" %}
+  {% endif %}
+
+  {% assign region = path | split: "/" | first | prepend: "/" %}
+  {% assign lastArg = path | split: "/" | last | prepend: "/"%}
+
+  {% assign outputStatus = false %}
+  {% assign emailUpdates = "https://www.va.gov/" %}
+  {% assign statusUrl = "https://www.va.gov/" %}
```

Comment:
I might be missing the point of this so forgive me in advance if that is the case.   By assigning the defaullt of the home page, aren't we going to output links that will fail to do what they say they are doing?  

Would it be better to assign them empty, then down in the output, check if empty and only output the buttons if they are not empty?

Reply:
Great catch, @swirtSJW - I was just copying a pattern without questioning it. Updated :+1: